<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editEntryDialog</class>
 <widget class="QDialog" name="editEntryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit entry</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblSite">
     <property name="text">
      <string>Site</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="txtSite"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblUsername">
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="txtUsername"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblEmail">
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="txtEmail"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblPassword">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="txtPassword"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblNotes">
     <property name="text">
      <string>Notes</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="txtNotes"/>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pbCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="pbAccept">
     <property name="text">
      <string>Accept changes</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
